.github/check_license.sh

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
```

Comment:
Don't we need a license on this file as well?

Reply:
Sure. But I don't think it is currently tested, and we should add it to the DLA-Future one as well.